### PR TITLE
CB-8992 Do not install mobilespec tests when --plugins specified

### PR DESCRIPTION
**Pull request by alsorokin**
https://issues.apache.org/jira/browse/CB-8992

### CODE REVIEW

**Comment by sgrebnov** [comment]
:1:

**Comment by dblotsky** [comment]
Why not install the mobilespec tests when the plugin is installed? Why is this undesired behaviour?

**Comment by sgrebnov** [comment]
If I explicitly specify list of plugins to be tested I don't want to run org.apache.cordova.mobilespec.tests (they represent some extra tests not related to our plugins like iframe, XHR, etc)

File: createmobilespec/createmobilespec.js [hunk 1]
```diff
@@ -56,7 +56,8 @@ function popd(dir) {
 }
 
 function pluginAdd(pluginName, searchPath, extraFlags) {
-    var command = cli + ' plugin add ' + pluginName + ' --searchpath ' + searchPath;
+    var sp = SEARCH_PATHS.hasOwnProperty(pluginName) ? SEARCH_PATHS[pluginName] : searchPath;
```
Comment (dblotsky) [inline createmobilespec/createmobilespec.js]: It's clearer to put this logic outside of the function, on [line 568] (https://github.com/apache/cordova-mobile-spec/pull/128/files#diff-ad19d116ad56f7b3ae8ee9fdc8733f69R568). That way pluginAdd is only concerned with adding a plugin from a given path, not with extra logic regarding the path.

File: createmobilespec/createmobilespec.js [hunk 2]
```diff
@@ -546,11 +547,12 @@ function installPlugins() {
         console.log("Installing local test framework plugins...");
         var linkPluginsFlag = (argv.link || argv.linkplugins) ? ' --link' : '';
 
-        pluginAdd('org.apache.cordova.mobilespec.tests', mobile_spec_git_dir, linkPluginsFlag + browserifyFlag);
+        if (!argv.plugins) {
```
Comment (dblotsky) [inline createmobilespec/createmobilespec.js]: Please comment why this check is here.

**Comment by dblotsky** [comment]
Ok. Sounds reasonable. I left some comments in review.

**Comment by alsorokin** [comment]
Thanks for the comments, Dmitry. I've addressed them and updated the PR.

**Pull request closed without merging**